Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env groovy
-
-/* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-buildPlugin()
+buildPlugin(useContainerAgent: true, configurations: [
+  [platform: 'linux', jdk: 11],
+  [platform: 'windows', jdk: 11],
```

Comment:
Requires someone like @olamy with write access to test in CI. Locally `mvn verify` passes with JDK 11.